docs/source/getting_started/changelog.rst

```diff
@@ -13,67 +13,66 @@ milestone on GitHub for the full list of pull requests and issues closed.
 User-facing changes
 ^^^^^^^^^^^^^^^^^^^
 
+Extensions can be installed without building JupyterLab with NodeJS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-
-Extensions are now Python packages and do not require ``node``
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-In JupyterLab 3.0, extensions are typically pip or conda packages that are dynamically loaded by the application (no more nodejs build step needed). See the `documentation <https://jupyterlab.readthedocs.io/en/latest/user/extensions.html#extensions>`__ for more details.
+In JupyterLab 3.0, a new recommended way of distributing and installing extensions as Python pip or conda packages is available. Installing such extensions does not require rebuilding JupyterLab and does not require having NodeJS installed. The previous way of distributing extensions as npm packages requiring rebuilding JupyterLab is still available as well. See the `documentation <https://jupyterlab.readthedocs.io/en/latest/user/extensions.html#extensions>`__ for more details.
 
 The JupyterLab interface is now translatable
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 JupyterLab now provides the ability to set the display language of the user interface.
-See the `documentation <https://jupyterlab.readthedocs.io/en/latest/user/language.html>`__ for more details.
+See the `internationalization documentation <https://jupyterlab.readthedocs.io/en/latest/user/language.html>`__ for more details.
 
 
-Added a visual debugger
-~~~~~~~~~~~~~~~~~~~~~~~
-JupyterLab now ships with a Debugger front-end by default.  See the `documentation <https://jupyterlab.readthedocs.io/en/latest/user/debugger.html>`__ for more details.
+A new visual debugger
+~~~~~~~~~~~~~~~~~~~~~
+
+JupyterLab now ships with a debugger front-end by default, available for kernels that support the new debugging protocol.  See the `documentation <https://jupyterlab.readthedocs.io/en/latest/user/debugger.html>`__ for more details.
 
 
 Improvements to Single Document Mode and Mobile 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
```

Comment:
The single-document user interface has been improved to be more streamlined. Both modes of JupyterLab now support showing the current file in use in the browser URL bar, similar to the classic notebook UI.